app/cdap/components/DrawerWidget/styles.ts

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { makeStyles } from '@material-ui/styles';
+
+export const useStyles = makeStyles(() => {
+  return {
+    paper: {
+      top: 46,
+      height: 'calc(100vh - 47px)',
+    },
+    drawerContainerStyles: {
+      width: 460,
+      height: '100%',
+      paddingLeft: 30,
+    },
+    headerStyles: {
+      height: 60,
+      display: 'flex',
+      justifyContent: 'space-between',
+      alignItems: 'center',
+      paddingLeft: 0,
+      paddingRight: 0,
+    },
+    headingStyles: {
+      display: 'flex',
+      flexDirection: 'column',
+      alignItems: 'flex-start',
+    },
+    pointerStyles: {
+      cursor: 'pointer',
+    },
+    chevronLeftRounded: {
+      fontSize: '24px',
```

Comment:
use number directly instead of number with px and check UI once after doing changes

Reply:
done